Assets/Scripts/Enums/Layers.cs

```diff
@@ -0,0 +1,30 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public static class Layers

```

Comment:
Kunnen we dit niet ophalen uit Unity zelf? Nu hebben we dubbel onderhoud

Reply:
We kunnen layers met naam ophalen, maar deze manier leek mij netter. Dit is misschien wel dubbel onderhoud, maar hoe vaak passen we onze layers aan? En in het gebruik is dit een stuk makkelijker dan steeds op te zoeken hoe die layer ook alweer precies heette.